blueprints/ember-backstop/files/ember-backstop/backstop_data/engine_scripts/puppet/overrideCSS.js

```diff
@@ -1,15 +1,17 @@
+/* eslint-env browser, node */
+
+const debug = require('debug')('PUPPET_ENGINE_SCRIPT');
 const BACKSTOP_TEST_CSS_OVERRIDE = `#ember-testing {width: 100% !important; height: 100% !important; -webkit-transform: scale(1) !important; transform: scale(1) !important;}`;
 
 module.exports = function(page, scenario) {
   // inject arbitrary css to override styles
-  page.evaluate(`window._styleData = '${BACKSTOP_TEST_CSS_OVERRIDE}'`);
   page.evaluate(() => {
-    var style = document.createElement('style');
+    let style = document.createElement('style');
     style.type = 'text/css';
-    var styleNode = document.createTextNode(window._styleData);
+    let styleNode = document.createTextNode(`${BACKSTOP_TEST_CSS_OVERRIDE}`);
```

Comment:
So, you are absolutely sure this works?  Yeah?   Just checking...  😅

Reply:
I tested it (using [ember-backstop-tutorial repo](https://github.com/garris/ember-backstop-tutorial)) as per our discussion over DM. Looked good to me.